packages/flutter/test/gestures/monodrag_test.dart

```diff
@@ -45,8 +45,14 @@ void main() {
     GestureBinding.instance.gestureArena.close(91);
     tester.async.flushMicrotasks();
 
-    GestureBinding.instance.handleEvent(up90, HitTestEntry(MockHitTestTarget()));
-    GestureBinding.instance.handleEvent(up91, HitTestEntry(MockHitTestTarget()));
+
+    // Regression test for https://github.com/flutter/flutter/issues/112403.
```

Comment:
I believe we don't need tests like this for null safety, it's considered "code refactor with no semantic change".
https://github.com/flutter/flutter/pull/114111#issuecomment-1292756760

https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
> refactors with no semantic change (e.g. null safety migrations)

However I believe @dkwingsmt's point in the last PR was: do we understand why is this null https://github.com/flutter/flutter/pull/117551#issuecomment-1363348820 ? If it's just a mistake from the null migration then this test can be removed.  However if it's a legitimate state to be in, can we exercise that state in a unit test?

Reply:
No I do not understand why this is null. I responded to an internal request for a 1p customer who had their pr closed because of a lack of tests. When I saw the pr was a null check crash and they were seeing this happen in the field I thought that it was worth timeboxing writing a test that triggers a null check.

Reply:
Someone else is welcome to take this pr over if they would like.